compas_python_utils/cosmic_integration/FastCosmicIntegration.py

```diff
@@ -802,14 +804,27 @@ def parse_cli_args():
     parser.add_argument("--delete", dest='delete_rates',
                         help="Delete the rate group from your hdf5 output file (groupname based on dP/dZ parameters)",
                         action='store_true', default=False)
+    parser.add_argument("--cosmology_name", dest='cosmology_name', help="Cosmology that is used for cosmic integration using the astropy.cosmology class: one of ['Planck13', 'Planck15', 'Planck18', 'WMAP1', 'WMAP3', 'WMAP5', 'WMAP9']", 
+                        type=str, default="Planck18")
 
     args = parser.parse_args()
     return args
 
+def set_cosmology(cosmology_name="Planck18"):
+    # Set cosmology using astropy, print a warning if TNG fit is used with Planck18 cosmology (since TNG uses Planck15)
+    if cosmology_name == "Planck18": print("USING PLANCK18 AS COSMOLOGY! If working with TNG fit, you may want to use Planck15 instead for self-consistency.")
+    else: print("Using %s as cosmology!"%cosmology_name)
+
+    return getattr(importlib.import_module('astropy.cosmology'), cosmology_name)
+
+
 
 def main():
     # Define command line options for the most commonly varied options
     args = parse_cli_args()
+    
+    cosmology = set_cosmology(cosmology_name=args.cosmology_name)
```

Comment:
Setting the cosmology here makes setting the cosmology via the python API a bit tricky, i suggest moving this line to the function that uses the cosmology. 

This way we can also enforce/type check that the `cosmology` parameter passed is always a string (atm i think the cosmology parameter passed in the `main()`  is an `astropy.cosmology` object but the default parameters in the function declaration is a string)